website/templates/socialaccount/connections.html

```
@@ -0,0 +1,36 @@
+{% extends "socialaccount/base.html" %}
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)